Stack_LinkedList.java

```diff
@@ -0,0 +1,91 @@
+	//Precourse 1: Exercise 2    --  Implementing stack using LinkedList
+	// Time Complexity : O(n) as traversal and insertion for n elements takes O(n) elements

```

Comment:
Same as previous, you are adding node to head of list and for removing you are just updating root pointer.